samples/RefreshFlow/AngularApp/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "typescript.tsdk": "./node_modules/typescript/lib",
+}
```

Comment:
The implicit flow sample already has VS Code-specific artifacts, so I guess it's probably fine to have them in the refresh token flow demo.